utils/api.hh

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
I don't understand why this file is called "api.hh"? We should name files based on their content, not where that content happened to live before.

Reply:
`utils/` in general is a place for low-level utility stuff that doesn't depend on any other parts of scylla. The content of this file however is higher level, as it works with database and tables.

Reply:
How about `service/utils.hh`?

Reply:
I don't think this code is worth trying to make reusable, it is so simple, you can just duplicate it.  Code reuse is most of the time great, but not every single time. Here it would lead to compaction code including stuff from api or service namespace, which is not desired.